packages/vite/src/node/plugins/resolve.ts

```diff
@@ -181,13 +180,6 @@ export interface ResolvePluginOptionsWithOverrides
 
 export function resolvePlugin(
   resolveOptions: ResolvePluginOptionsWithOverrides,
-  /**
-   * @internal
-   * config.createResolver creates a pluginContainer before environments are created.
-   * The resolve plugin is especial as it works without environments to enable this use case.
-   * It only needs access to the resolve options.
-   */
-  environmentsOptions?: Record<string, ResolvedEnvironmentOptions>,
```

Comment:
I checked `config.createResolver`, but didn't find this to be used. Is this still used?